client/landing/jetpack-cloud/components/log-item/index.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React, { ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import CardHeading from 'components/card-heading';
```

Comment:
Nit: could we 🔤 alphabetize these imports?

Reply:
Resolved in 788beea0bdb4691b0c9f344152a4268d5571095f.